### PR TITLE
consolidate excel gsheets

### PR DESCRIPTION
**Pull request by mirnawong1**
this pr adds saved queries and saved selection support for the SL excel integration. 

it also consolidates the excel and gsheets content into one snippet

### CODE REVIEW

File: website/snippets/_sl-excel-gsheets.md [hunk 1]
```diff
@@ -65,26 +65,24 @@
...
-#### Other settings 
+#### Other settings
 
 <p>If you would like to just query the data values without the headers, you can optionally select the <strong>Exclude Column Names</strong> box.</p>
-<p>To return your results and keep any previously selected data below it intact, un-select the <strong>Exclude Column Names</strong> box. By default, we'll clear all trailing rows if there's stale data. </p>
-
-
-
-<!-- hide this content for initial launch
+<p>To return your results and keep any previously selected data below it intact, un-select the <strong>Exclude Column Names</strong> box. By default, we'll clear all trailing rows if there's stale data.</p>
```
Comment (rpourzand) [inline website/snippets/_sl-excel-gsheets.md]: I think I might of had a typo.. this should be "Clear Trailing Rows" not "Exclude Column Names"

Comment (rpourzand) [reply website/snippets/_sl-excel-gsheets.md]: There's both exclude column names and clear trailing rows, it looks like it's repeated.

Comment (mirnawong1) [reply website/snippets/_sl-excel-gsheets.md]: good shout! have fixed this now

File: website/docs/docs/cloud-integrations/semantic-layer/gsheets.md [hunk 2]
```diff
@@ -40,13 +40,15 @@ import Tools from '/snippets/_sl-excel-gsheets.md';
...
 />
 
-<!-- adding this section here temporarily (not incl limited policy) until saved queries is available for excel. when saved queries becomes available for excel365 integration:
-1. remove the below content (tip and using saved queries header
-2. then go to the website/snippets/_sl-excel-gsheets.md snippet
-3. And uncomment line 97 - 118 to make saved queries content available to gsheets AND excel -->
+<!-- this comment explains the following:
+1. uses the /snippets/_sl-excel-gsheets.md snippet file to render gsheets and excel content since they're similar
+2. removes the below content (tip and using saved queries header) so it's not duplicative. removing the below content fully a month or so after the launch.
+3. keep tje limited policy section in gsheets only and do not add to the /snippets/_sl-excel-gsheets.md snippet file.
```
Comment (nataliefiann) [inline website/docs/docs/cloud-integrations/semantic-layer/gsheets.md:51]: Hiya @mirnawong1 

This is a non-blocker (I think) as it appears to have been removed, but I wasn't able to find this section (lines 48 - 51) on the staging preview page).

Kind Regards
Natalie

**Review by nataliefiann — APPROVED** [review]
Hiya @mirnawong1 

Thank you for creating this PR - it's great! I left a comment but I think it may have already been resolved (but thought i'd raise it just in case I missed it). Other than that, there are no suggestions or recommendations from me.

Kind Regards

**Comment by mirnawong1** [comment]
merging once i get the final ok from @rpourzand or @Jstein77

**Review by Jstein77 — APPROVED** [review]
LGTM!

**Pull request merged**